js/views/itemEditVw.js

```diff
@@ -55,7 +56,16 @@ module.exports = Backbone.View.extend({
     self.model.set("imageHashesToUpload", anotherHashArray);
     self.model.set('expTime', self.model.get('vendor_offer').listing.metadata.expiry.replace(" UTC", ""));
 
-    this.render();
+    // prevent the body from picking up drag actions
+    $(document.body).on("dragover", this.onDragonover = function(e) {
+      e.preventDefault();
+      return false;
+    });
```

Comment:
Isn't this already being done when the app starts up, or did it accidentally get removed?